rotkehlchen/accounting/structures.py

```diff
@@ -122,6 +122,16 @@ def __add__(self, other: Any) -> 'Balance':
             usd_value=self.usd_value + other.usd_value,
         )
 
+    def __radd__(self, other: Any) -> 'Balance':
```

Comment:
During this feature I've used `sum([<Balance>, ...])` but for having to fight less with mypy and for make it a bit more readable I've discarded in pro of `sum([FVal, ...])` (via `<Balance>.usd_value` or <Balance>.amount`). I've decided to keep these radd implementations as they've proved to be useful (I've included their tests).